### PR TITLE
bazel 0.1.5

### PR DESCRIPTION
**Pull request by damienmg**

### CODE REVIEW

**Comment by dfabulich** [comment]
There's something very wrong with this formula. I tried testing it out locally, and it doesn't install `bazel` in `/usr/local/bin`.

```
$ brew install bazel
==> Downloading https://homebrew.bintray.com/bottles/bazel-0.1.5.el_capitan.bottle.tar.gz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "bazel"
Download failed: https://homebrew.bintray.com/bottles/bazel-0.1.5.el_capitan.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Downloading https://github.com/bazelbuild/bazel/archive/0.1.5.tar.gz
Already downloaded: /Library/Caches/Homebrew/bazel-0.1.5.tar.gz
==> ./compile.sh
==> ./output/bazel build scripts:bash_completion
==> Caveats
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d

zsh completion has been installed to:
  /usr/local/share/zsh/site-functions
==> Summary
🍺  /usr/local/Cellar/bazel/0.1.5: 7 files, 95K, built in 3 minutes 27 seconds
dfab: /usr/local $ which bazel
dfab: /usr/local $ ls /usr/local/bin/bazel
ls: /usr/local/bin/bazel: No such file or directory
```

File: Library/Formula/bazel.rb [hunk 1]
```diff
@@ -23,13 +23,7 @@ def install
     system "./compile.sh"
     system "./output/bazel", "build", "scripts:bash_completion"
 
-    (prefix/"base_workspace").mkdir
-    cp_r Dir["base_workspace/*"], (prefix/"base_workspace"), :dereference_root => true
-    bin.install "output/bazel" => "bazel"
```
Comment (dfabulich) [inline Library/Formula/bazel.rb]: This `bin.install` line needs to remain, or else nothing will be installed in `/usr/local/bin`.

**Comment by damienmg** [comment]
Sorry for the error, updated the PR

File: Library/Formula/bazel.rb [hunk 2]
```diff
@@ -23,13 +23,8 @@ def install
...
     system "./output/bazel", "build", "scripts:bash_completion"
 
-    (prefix/"base_workspace").mkdir
-    cp_r Dir["base_workspace/*"], (prefix/"base_workspace"), :dereference_root => true
     bin.install "output/bazel" => "bazel"
     (prefix/"etc/bazel.bazelrc").write <<-EOS.undent
-      build --package_path=%workspace%:#{prefix}/base_workspace
-      query --package_path=%workspace%:#{prefix}/base_workspace
-      fetch --package_path=%workspace%:#{prefix}/base_workspace
     EOS
```
Comment (bfontaine) [inline Library/Formula/bazel.rb]: You can remove these lines as well:
```rb
(prefix/"etc/bazel.bazelrc").write <<-EOS.undent
EOS
```

Comment (damienmg) [inline Library/Formula/bazel.rb]: Oh yes because it is installed on a separate folder... Will do

**Comment by damienmg** [comment]
Removed.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/25a842faa620d09bfd5b7ce2c3a23e5d69cf07ca. Thank you for another contribution to Homebrew @damienmg! :smiley_cat:

**Pull request closed without merging**